### DISCUSSION TITLE
Kit supports 24 hours to receive

### DISCUSSION THREAD

**Discussion by XXY233** [Zrips/CMI#3316] [Feature Requests]
**Description of feature/improvement request:**
That is, you can get kit once a day, not calculated according to the cooling time.

---
### Detailed explanation of what it should do, when and in what situation: ###
You can customize days to receive a kit instead of according to the cooling time.

### Basic explanation as to why this feature is needed and/or useful to have: ###
This is very friendly to players, so you do n’t have to wait too long every day to pick up the kit.

**Comment by Zrips** [comment]
But then why not simply use cooldown of something like 22 hours on a kit? Player can get it every day without any trouble, and you will not really have anyone who would try to get one extra kit by picking one every 22 hours exactly, which would include in a middle of night at some point. Unless i'm missing entire point here.

**Comment by XXY233** [comment]
Because I like CMI so much, I do n’t want to use other plugins after using CMI. I made the check-in kit with cmi, which is available after 00:00 every day or 00:00 after a few days.
If the previous day was used late at night, then the next day does not have to wait until late at night to use.
I also hope to increase the first time every day to go online to automatically execute commands for players, or players with permissions,this can do more things, such as daily welcome.
(Because the player's backpack may be full, the above kit still needs to exist in order to use it again.)
